### DISCUSSION TITLE
unrecognized parameter: "Base_error_model.txt"

### DISCUSSION THREAD

**Discussion by akaraw** [hahnlab/CAFE5#77] [Q&A] [answered]

**Comment by kataksk** [comment]
I also encountered the same error. I ran CAFE5 with the Base_error_model.txt file that I got as a result of running with the -e parameter . Wondering to know about what has happened to this issue since then?

**Comment by benfulton** [comment]
What command line did you use?

**Reply by kataksk** [reply, marked as answer]
It has been solved! I checked the manual again with "CAFE5 -h", and found there is no need to provide a space between "-e" and file path. Thanks a lot.

-e: Run with no file name to estimate the global error model file. This file can be provided in subsequent runs by providing the path to the Error model file with no spaces (**_e.g. -eBase_error_model.txt_**).
